### PR TITLE
Initial set-up for ReadtheDocs documentation for Perseo Core.

### PR DESCRIPTION
**Pull request by jason-fox**
- Please create a project on readthedocs called fiware-perseo-core and point to this repository.
- Includes standard CSS and badges.
- Just a dump of existing documentation but with CSS styling.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+site_name: Perseo Core
+site_url: https://fiware-perseo-core.readthedocs.org
+repo_url: https://github.com/telefonicaid/perseo-dore.git
```
Comment (fgalan) [inline mkdocs.yml]: Typo: perseo-dore -> perseo-core

**Comment by fgalan** [comment]
Proyecct just created in RTD. Waiting for typo fixing in the PR to merge.

File: mkdocs.yml (hunk 1 above)
Comment (jason-fox) [reply mkdocs.yml]: Fixed 7940dd9

**Comment by fgalan** [comment]
I think this PR was waiting for the bigger one in Perseo-FE (PR https://github.com/telefonicaid/perseo-fe/pull/330, finally moved to PR  https://github.com/telefonicaid/perseo-fe/pull/592)

Now that one is merged, what should we do with this PR?

**Pull request closed without merging**